AdyenCard/Components/Card/CardComponentConfiguration.swift

```diff
@@ -33,11 +46,10 @@ extension CardComponent {
         /// Indicates whether to show the security code field at all. Defaults to true.
         public var showsSecurityCodeField: Bool
 
-        /// Indicates whether to show the security fields for South Korea issued cards. Defaults to false.
-        public var showsKoreanAuthentication: Bool
+        /// Indicates whether to show the security fields for South Korea issued cards. Defaults to .auto.
+        public var showsKoreanAuthentication: FieldMode
```

Comment:
maybe we can change the name of this, since `shows...` suggests a boolean value

Reply:
👍